pkgloader-recommended.sty

```diff
@@ -175,6 +175,25 @@
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  A list of recommended rules extracted from the uri
+%  documentation and uses of |\@ifpackageloaded| and |\RequirePackage| in
+%  |uri.sty|:
+%
+%    \begin{macrocode}
+\Load {hyperref} before {uri}
+                 if loaded
+                 because {uri uses hyperref's commands if hyperref is loaded. See also Section 2 of uri.pdf.}
+\Load {kvoptions} before {uri}
+                  if loaded
+                  because {uri has a RequirePackage on kvoptions}
```

Comment:
I don't think this rule does anything? `uri` already loads `kvoptions`. And `kvoptions` doesn't take any options itself. `url` accepts an option and `uri` doesn't provide any, so I see the reasoning for that one.